backstory/src/main/java/com/google/sps/story/StoryManagerImpl.java

```diff
@@ -45,6 +45,15 @@
   /** requestFactory - Builds and facilitates authenticated post requests. */
   private StoryManagerRequestFactory requestFactory;
 
+  /** serviceUrls - URLs for each story generation container */
+  private String[] serviceUrls = {"https://backstory-text-gen-1-pdaqhmzgva-uc.a.run.app",
```

Comment:
According to the Java style guide, I think that this array is supposed to be formatted slightly differently. E.g. the first String should not be on the same line as serviceUrls. However, if you just add a comma after the last String, then `make pretty` will format it for you the way the Java style guide does it.